.github/release-drafter.yml

```diff
@@ -0,0 +1,19 @@
+name-template: "v$NEXT_PATCH_VERSION 🌈"
+tag-template: "v$NEXT_PATCH_VERSION"
+categories:
+  - title: "🚀 Features"
+    labels:
```

Comment:
Does it mean issue label? or PR ?

Reply:
Supposed to be pr labels

Reply:
We could try to edit the label on this pr too if it's to be merged, and see what will it do :)